bin/deploy_kubevirt_vm

```diff
@@ -365,69 +365,29 @@ create_namespace_and_secret(){
 create_datavolume_and_pvc() {
     log_timestamp_console "Creating DataVolume and PVC..."
 
-    for ((i=1; i<=3; i++)); do
-        log_timestamp "Attempt $i to create DataVolume and PVC..."
+    log_timestamp "Applying DataVolume CR..."
+    create_datavolume_cr
+    check_command "Failed to create_datavolume_cr." || continue
 
-        log_timestamp "Applying DataVolume CR..."
-        create_datavolume_cr
-        check_command "Failed to create_datavolume_cr." || continue
-
-        log_timestamp "Applying DataVolume 2 CR..."
-        create_datavolume_cr_2
-        check_command "Failed to create_datavolume_cr_2." || continue
-
-        log_timestamp_console "Waiting for data volume to be bound..."
-        local timeout=2400
-        local start_time=$(date +%s)
-        while true; do
-            pvc_status=$(oc get pvc $VM_NAME -n $NAMESPACE -o=jsonpath='{.status.phase}')
-            db_pvc_status=$(oc get pvc $VM_NAME-database -n $NAMESPACE -o=jsonpath='{.status.phase}')
-
-            log_timestamp "PVC status: $pvc_status, Database PVC status: $db_pvc_status"
-
-            if [[ "$pvc_status" == "Bound" && "$db_pvc_status" == "Bound" ]]; then
-                log_timestamp "PVCs are bound successfully."
-                return 0
-            fi
-
-            if [ $(($(date +%s) - start_time)) -ge $timeout ]; then
-                log_error_timestamp "PVC binding timed out after ${timeout} seconds."
-                break
-            fi
-            sleep 5
-        done
-
-        log_timestamp_console "PVCs are not bound. Cleaning up and retrying..."
-        oc delete pvc $VM_NAME -n $NAMESPACE
-        oc delete pvc $VM_NAME-database -n $NAMESPACE
```

Comment:
Do we need the cleanup code in case of pvc binding timeout?

Reply:
The old code didn't really timeout per se, it just didn't wait long enough. Instead it had an artificial timeout and then would just blow away things (even if they were in the middle of importing.